app/Http/Controllers/BundleController.php

```diff
@@ -15,113 +15,119 @@ class BundleController extends Controller
 {
 
 	// The bundle content preview
-	public function previewBundle(Request $request, Bundle $bundle) {
+	public function previewBundle(Request $request, Bundle $bundle)
+	{
 		return view('download', [
 			'bundle'		=> new BundleResource($bundle)
 		]);
-
 	}
 
 	// The download method
 	// - the bundle
 	// - or just one file
-	public function downloadZip(Request $request, Bundle $bundle) {
+	public function downloadZip(Request $request, Bundle $bundle)
+	{
 
-		try {
-			// Download of the full bundle
-			// We must create a Zip archive
-			$bundle->downloads ++;
-			$bundle->save();
+		// try {
+		// Download of the full bundle
+		// We must create a Zip archive
+		$bundle->downloads++;
+		$bundle->save();
 
 
-			$filename	= Storage::disk('uploads')->path('').'/'.$bundle->slug.'/bundle.zip';
```

Comment:
Instead of removing the `disk('uploads')` everywhere, wouldn't it be better to add a .env setting to choose the uploads disk storage driver? Something like this: 
```
        'uploads' => [
            'driver' => env('UPLOAD_STORAGE_DRIVER', 'local'),
            'root'  => storage_path('content/uploads'),
			'visibility' => 'private',
			'permissions' => [
				'file' => [
					'public' => 0600,
					'private' => 0600,
				],
				'dir' => [
					'public' => 0755,
					'private' => 0700,
				],
			]
        ],
```
In the case, default storage remains local, and upload files are stored on the defined driver.